homeworks/09-node-chokidar/modules/dirWatcher.ts

```diff
@@ -0,0 +1,53 @@
+import EventEmitter from 'events';
+import { readdirSync } from 'fs';
+
+export const CHANGED_EVENT = 'dirwatcher:changed';
+
+export type DirWatcherConfig = {
+  dirPath: string,
+  fileExtention: string,
+  watchInterval: number,
+};
+
+export class DirWatcher {
+  private watchedFileNames: string[];
+
+  constructor(
+    public eventEmitter: EventEmitter,
+    public config: DirWatcherConfig,
```

Comment:
i guess that providing getter for config would be better that it's public. It'd be great to make the config only for reading.

Reply:
added in https://github.com/vbarbasins/evo-ts-bootcamp/pull/10/commits/37e95faf8f9b2447ff9b0b2293cec8e82c681932